File: condition_and_loop/find_number.py
#  Write a Python program to find those numbers which are divisible by 7 and multiple of 5, between 1500 and 2700

def find_numbers(init, finish):
    numbers = []
    for i in range(init, finish+1):
        if (i % 7 == 0) and (i % 5 == 0):
            numbers.append(str(i))

    print("-".join(numbers))
    # print(numbers)

find_numbers(1500, 2700)

File: ruby_and_python/2_condition_and_loop/find_number.rb
#  Write a Python program to find those numbers which are divisible by 7 and multiple of 5, between 1500 and 2700



def find_numbers(init, final)

  numbers = Array.new


  init.upto(final) do |i|
    if (i%7 == 0) && (i%5 == 0)
      numbers << i
    end
  end
    puts numbers.join("-")
end


find_numbers(1500, 2700)
